Ginger/Ginger/SolutionWindows/TargetApplicationsPage.xaml.cs

```diff
@@ -62,16 +63,21 @@ private void SetAppsGrid()
             xTargetApplicationsGrid.SetGridEnhancedHeader(Amdocs.Ginger.Common.Enums.eImageType.Application, "Target Applications", saveAllHandler: SaveHandler, addHandler: AddApplication);
             GridViewDef view = new GridViewDef(GridViewDef.DefaultViewName);
             view.GridColsView = new ObservableList<GridColView>();
+            //List<string> platformesTypesList = GingerCore.General.GetEnumValues(typeof(ePlatformType));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5700269)